config_utilities/src/asl_formatter.cpp

```diff
@@ -63,9 +63,12 @@ std::string AslFormatter::formatErrorsRecursive(const MetaData& data, const std:
   const std::string name_prefix_before = name_prefix_;
   if (Settings::instance().inline_subconfig_field_names) {
     if (!data.field_name.empty()) {
+      // TOOD(nathan) refactor to put in metadata
```

Comment:
I think this is purely a printing problem? Not sure if this needs to go into meta data.

Reply:
yeah, I wasn't sure what the right split between formatting the indices/keys and just getting the string values of the indices/keys if they exist was so I punted on figuring it out for now